src/sbw.erl

```diff
@@ -214,9 +215,9 @@ headers(Wrapper) ->
 
 header(Header, Wrapper) ->
     BinHeader = simple_bridge_util:binarize_header(Header),
-    case lists:keyfind(BinHeader, 1, Wrapper#sbw.headers) of
-        false -> undefined;
-        {_, Val} ->
+    case maps:find(BinHeader, Wrapper#sbw.headers) of
+        error -> undefined;
+        {ok, {_Key,Val}} ->
```

Comment:
Here, we *should* be able to just  do `{ok, Val}` as referenced on line 121 above.

Reply:
ok